appveyor.yml

```diff
@@ -21,5 +21,5 @@ install:
 
 build_script:
   - go test -tags="testing debug" -short ./...
-  - go test -tags="testing debug" -v -race ./...
-  - go test -tags="testing debug" -v -timeout=500s -run=XXX -bench=. ./...
+  - go test -tags="testing debug" -v -timeout=1200s -race ./...
```

Comment:
travis I know will time-out if there's no output for 10 minutes, and I also know for some reason the test package won't always flush the buffers inside of 10 minutes if you have a bunch of tests running in parallel. Not sure why, but this could impact appveyor somehow as well. I guess we'd need to refer to the appveyor documentation.